opensrp-reveal/src/main/java/org/smartregister/reveal/widget/RevealEditTextFactory.java

```diff
@@ -57,4 +65,24 @@ protected int getLayout() {
             return super.getLayout();
         }
     }
+
+    private void addRelativeNumericIntegerValidator(JSONObject editTextJSONObject, JsonFormFragment formFragment, MaterialEditText editText) throws JSONException {
```

Comment:
Move this method call to  nativeform EditTextFactory class

Reply:
Done.